Resources/Public/Scripts/SurfCaptainApp/Controllers/DeployController.js

```diff
@@ -118,8 +118,7 @@
             for (property in presets) {
                 if (presets.hasOwnProperty(property)) {
                     if (angular.isUndefined(presets[property].applications[0].type) ||
-                        presets[property].applications[0].type === CONFIG.applicationTypes.deployTYPO3 ||
-                        presets[property].applications[0].type === CONFIG.applicationTypes.deploy) {
+                        presets[property].applications[0].type.indexOf('Deploy') >= 0) {
```

Comment:
OK for now. But better would be to have tis check in PresetService or UtilityService or ConfigurationService or something like this:

```
if (PresetService.isPresetDeployable(preset)) {
```